services/dominant/values.yaml

```
@@ -0,0 +1,51 @@
+image:
+  repository: dr2.rbkmoney.com/rbkmoney/dominant
+  pullPolicy: IfNotPresent
+
+imagePullSecrets:
+  name: dr2reg
+
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: false
```

Может
```suggestion
  create: true
```

чтобы как в других чартах было